docs/docs/administration/install-via-docker-compose.md

```diff
@@ -110,7 +110,7 @@ Restart the docker containers for the configuration to take effect.
 If you'd like to use an external PostgreSQL server for Mathesar's internal database, you'll need to do the following:
 
 
-1. On the existing database server, [create a new database](https://www.postgresql.org/docs/current/sql-createdatabase.html) for Mathesar to store its metadata.
+1. On the external PostgreSQL server, [create a new database](https://www.postgresql.org/docs/current/sql-createdatabase.html) for Mathesar to store its metadata.
```

Comment:
I think this change makes things more, rather than less, confusing. I.e., "external PostgreSQL server for Mathesar's internal database..."

Reply:
I've removed the term `existing/external` and used `your PostgreSQL server` instead. I think this would be more clear.

Here's the commit: https://github.com/mathesar-foundation/mathesar/pull/4191/commits/78095c78c8b522a7b31a86c5e08f1cefb0ab4f8e